lineapy/transformer/node_transformer.py

```diff
@@ -187,19 +187,6 @@ def visit_NameConstant(self, node: ast.NameConstant) -> LiteralNode:
         else:
             return self.tracer.literal(node.value, self.get_source(node))
 
-    # FIXME - this is deprecated
```

Comment:
Just for my curiosity, how is this thing used before?

Reply:
check out callnode visitor, we check if argument is instance of ast.starred and do a different behavior if it is (add a flag to ensure we expand the starred var's value). It is more general than trying to map our every single var type that can be passed (what we were trying to do here).